bits/expiring_certificate/expiring_certificate.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+from typing import Iterator, List
+
+from octopoes.models import OOI
+from octopoes.models.ooi.certificate import Certificate
+from octopoes.models.ooi.findings import KATFindingType, Finding
+from octopoes.models.ooi.web import Website
+
+THRESHOLD = datetime.timedelta(weeks=2)
+
+
+def run(input_ooi: Certificate, additional_oois: List[Website]) -> Iterator[OOI]:
+    # applies only to certificates assigned to websitess
```

Comment:
It should apply to all objects, no reason to not include other certs. However, we should only produce a finding for certificates that we see in use somewhere.

Reply:
Good point. See my other comment below. I rephrased this specific comment on line 13